### ISSUE TITLE
1.13.1 updates 

### ISSUE THREAD

**Issue by Vestle** [kicjow/Crazy-Enchantments#341]
Hi there! Just letting you know some current bugs include Gui all items are stone. When a book is purchased it is also stone and will not go onto items. It still thinks it is an enchant but it doesn't work. Many console errors for the items with ce on them. Hope this may help in detecting certain problems for a 1.13.1 update. Let me know if I can assist in update testing.

**Comment by Vestle** [comment]
Will donate asap for a update!

**Comment by pacopc555** [comment]
The same for me, and I can´t edit the item number id to a "minecraft:" id

**Comment by Vestle** [comment]
UPDATE: No need for a new version! Please note:
1. Change any ID to ALL CAPS in 'single_quotes'
Ex. ID 278 = diamond pickaxe - - > 1.13: 'DIAMOND_PICKAXE'

2. Make sure to edit Messages.yml as there are IDs in there as well.

3. When making gkitz be sure to use ALL CAPS. Do NOT list the catagory title for Enchantments: or CustomEnchantments:
Ex. - 'Item:DIAMOND_SWORD, Name:Custom Sword, Viper:1, Blindness:2'

**Comment by pacopc555** [comment]
I refer to for example red glass

**Comment by Vestle** [comment]
RED_STAINED_GLASS

Panes: RED_STAINED_GLASS_PANE

**Comment by FutureLights** [comment]
Would either of you mind sharing your 1.13 compatible config? I can't seem to get mine to work.

**Comment by nerfchase** [comment]
[Half Working CE Config.zip](https://github.com/badbones69/Crazy-Enchantments/files/3065682/Half.Working.CE.Config.zip)

I managed to get all but the unclickable stone around the options to change. I changed the glass to shulker boxes of respective colors. (I did change the economy costs to XP Levels, so you'll have to change that if you want it back.) Sorry I can't get more to work. Thought anything was better than nothing.
